dev/ci/ci-common.sh

```diff
@@ -111,11 +111,11 @@ git_download()
   if [ -d "$dest" ]; then
     echo "Warning: download and unpacking of $project skipped because $dest already exists."
   elif [[ $ov_url ]] || [ "$WITH_SUBMODULES" = "1" ] || [ "$CI" = "" ]; then
-    git clone "$giturl" "$dest"
+    git clone --depth 1 --branch "$ref" "$giturl" "$dest"
```

Comment:
ref is not always a branch, on Coq's release branch it gets set to a commit

Reply:
Ah. The man page says `--branch` can also take tags, but I haven't found a reliable way to request a specific commit. So if we want this change at all, it'll need to fall back to clone-and-switch.

Reply:
Actually, `init` and `pull` seem to do the trick for the repositories currently in CI.